app/code/Magento/Sales/Test/Unit/Controller/Adminhtml/Order/Invoice/SaveTest.php

```diff
@@ -98,11 +130,38 @@ protected function setUp(): void
         $contextMock->expects($this->any())
             ->method('getMessageManager')
             ->willReturn($this->messageManagerMock);
+        $this->objectManager = $this->createPartialMock(
+            FrameworkObjectManager::class,
+            ['create','get']
+        );
+        $contextMock->expects($this->any())
+            ->method('getObjectManager')
+            ->willReturn($this->objectManager);
+        $registryMock = $this->createMock(Registry::class);
+        $this->invoiceSender = $this->getMockBuilder(InvoiceSender::class)
+            ->disableOriginalConstructor()
+            ->setMethods(['send'])
+            ->getMock();
+        $this->invoiceSender->expects($this->any())
+            ->method('send')
+            ->willReturn(true);
+        $this->salesData = $this->getMockBuilder(SalesData::class)
+            ->disableOriginalConstructor()
+            ->setMethods(['canSendNewInvoiceEmail'])
+            ->getMock();
+        $this->invoiceService = $this->getMockBuilder(InvoiceService::class)
+            ->disableOriginalConstructor()
+            ->setMethods(['prepareInvoice'])
+            ->getMock();
 
         $this->controller = $objectManager->getObject(
             Save::class,
             [
                 'context' => $contextMock,
+                'registry' => $registryMock,
```

Comment:
please just remove it, it's not actually used and causing MHI test failure that will require additional approval.